OpenEdXMobile/src/main/java/org/edx/mobile/view/AccountFragment.java

```diff
@@ -72,4 +82,20 @@ public void onClick(View v) {
 
         return binding.getRoot();
     }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+        final Bundle arguments = getArguments();
+        if (arguments != null) {
+            @ScreenDef String screenName = arguments.getString(Router.EXTRA_SCREEN_NAME);
+            if (!TextUtils.isEmpty(screenName)) {
+                switch (screenName) {
```

Comment:
if there is no other case, then we have to use the if statement instated of switch.

Reply:
done